manager/volume.go

```diff
@@ -178,6 +178,9 @@ func (m *VolumeManager) Create(name string, spec *types.VolumeSpec) (v *longhorn
 		if baseImage, ok := backup.Labels[types.BaseImageLabel]; ok {
 			spec.BaseImage = baseImage
 		}
+		if spec.Standby == false {
+			spec.RestorationRequired = true
```

Comment:
This logic shouldn't be here. It's the internal logic of the controller. Here we simply set volume spec according to the input.